app/controllers/root_controller.rb

```diff
@@ -0,0 +1,6 @@
+# app/controllers/root_controller.rb
+class RootController < ApplicationController
+  def index
+    @random_greeting = Greeting.order('RANDOM()').first
+  end
```

Comment:
- [ ] Hello, kindly note that this action does not need to perform any database query